src/components/Property/RaiseWorkOrder/TradeContractorRateScheduleItemView.js

```diff
@@ -88,7 +88,15 @@ const TradeContractorRateScheduleItemView = ({
     }
   }
 
-  const incrementalSORSearchRequired = async (contractorRef, tradeCode) => {
+  useEffect(() => {
+    console.info('On load - checking if incremental search is required', {
```

Comment:
Do you want to keep this console output?

Reply:
It was useful. maybe not anymore